.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Packer Community Support
+    url: https://www.packer.io/community
```

Comment:
Yes, I thought the community page was a good place to redirect to, as it shows additional links that are interesting, but you're right, we should probably directly link to Discuss here, since the intent is already to ask a question, so the current proposed solution will add an extra step for basically nothing.
I'll change this, thanks!